manifests/repo/debian.pp

```diff
@@ -72,17 +72,17 @@
     }
   }
 
-  if ($sury and $php::globals::php_version == '7.1') {
-    # Required packages for PHP 7.1 repository
+  if ($sury and $php::globals::php_version in ['5.6','7.1','7.2'] ) {
+    # Required packages for PHP sury repository
     ensure_packages(['lsb-release', 'ca-certificates'], {'ensure' => 'present'})
 
-    # Add PHP 7.1 key + repository
-    apt::key { 'php::repo::debian-php71':
+    # Add PHP sury key + repository
+    apt::key { 'php::repo::debian-php-sury':
       id     => 'DF3D585DB8F0EB658690A554AC0E47584A7A714D',
       source => 'https://packages.sury.org/php/apt.gpg',
     }
 
-    ::apt::source { 'source_php_71':
+    ::apt::source { 'source_php_sury':
```

Comment:
Is that the name of the created file in `/etc/apt/sources.list.d/`? So people would end up with two files that contain the same repo?

Reply:
I think it's a good idea to remove the "7.1" mark in the repo name. Sury is stable and will be used too for future versions.